Sources/Quick/QuickSpec.swift

```diff
@@ -0,0 +1,29 @@
+import XCTest
+
+// NOTE: This file is not intended to be included in the Xcode project. It
+//       is picked up by the Swift Package Manager during its build process.
+
+public class QuickSpec: XCTestCase, XCTestCaseProvider {
```

Comment:
I didn't notice this on my first pass, but this could use documentation. Perhaps include the latest snapshot we've tested against?